function5.js

```diff
@@ -1 +1,12 @@
 // Create a function which will coordinates of 2 points. Each cordinate has x and y keys. Funxtion should return difference between these points as object {x: ?, y: ?}
+
+const getCoordinates = (x, y) => {
```

Comment:
Not really

1 coordinates is [x, y]

your function should get 2 coordinates.